### PR TITLE
CRW-479 Fixing installation with '--tls --self-signed-cert' options

### PR DESCRIPTION
**Pull request by ibuziuk**
### What does this PR do?
Fixing installation with '--tls --self-signed-cert' options

### What issues does this PR fix or reference?
https://issues.jboss.org/browse/CRW-479

Tested via `./run server:start -p crc --tls --self-signed-cert`

### CODE REVIEW

**Comment by skabashnyuk** [comment]
is this somehow backported from upstream? Or it has some CRW specific?

File: src/api/kube.ts [hunk 1]
```diff
@@ -954,11 +954,9 @@ export class KubeHelper {
 
       yamlCr.spec.auth.openShiftoAuth = flags['os-oauth']
       yamlCr.spec.server.tlsSupport = flags.tls
-      if (flags.tls) {
-        yamlCr.spec.k8s.tlsSecretName = 'che-tls'
```
Comment (davidfestal) [inline src/api/kube.ts]: Wouldn't it be codeready-tls ? Is it expected to fully remove the value ?

Comment (ibuziuk) [reply src/api/kube.ts]: this property is k8s specific, so for openshift we do not need them - https://github.com/eclipse/che-operator/blob/master/deploy/crds/org_v1_che_cr.yaml#L108-L122

**Comment by ibuziuk** [comment]
@skabashnyuk those props are k8s specific https://github.com/eclipse/che-operator/blob/master/deploy/crds/org_v1_che_cr.yaml#L108-L122 and not needed for CRW (since it is openshift only) - https://github.com/eclipse/che/blob/f26ab69c0d290c955e4abd159094a942788d9a70/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/IngressTlsProvisioner.java#L53

I discussed it with @sleshchenko

**Comment by skabashnyuk** [comment]
https://github.com/redhat-developer/codeready-workspaces-chectl/ is this a fork of https://github.com/che-incubator/chectl/ that is not going to be updated with a newer version?

**Comment by ibuziuk** [comment]
@skabashnyuk https://github.com/redhat-developer/codeready-workspaces-chectl/  is based on `7.3.x` branch of https://github.com/che-incubator/chectl/ (it was updated today for example after 7.3.2 release)

**Comment by skabashnyuk** [comment]
> https://github.com/redhat-developer/codeready-workspaces-chectl/ is based on 7.3.x branch of https://github.com/che-incubator/chectl/ (it was updated today for example after 7.3.2 release)

ok. For now. How are you going to do that next time with version > 7.4 and above? Reviewing by each change one by one?

**Comment by ibuziuk** [comment]
> How are you going to do that next time with version > 7.4 and above? Reviewing by each change one by one?

@skabashnyuk CRW 2.0 is based on 7.3.x, so we do not care about 7.4 (relevant changes are simply backported to 7.3.x)

**Pull request merged**